packages/config/config/devices/0x000c/hs-wx300.json

```
@@ -161,6 +161,81 @@
 					"value": 1
 				}
 			]
+		},
+		{
+			"#": "33",
+			"label": "Startup Flash Behavior",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "LEDs do not flash on startup",
+					"value": 0
+				},
+				{
+					"label": "LEDs flash on startup to indicate switch or dimmer operation",
+					"value": 1
+				}
+			]
```

```suggestion
			"$import": "~/templates/master_template.json#base_enable_disable",
			"label": "Indicate Operation Mode On Startup"
```
(I hope this is what the parameter is supposed to do, I wasn't quite sure)